docs/content/users/quickstart.md

```diff
@@ -8,14 +8,45 @@ Before proceeding, make sure your installation of DDEV is up to date. In a new a
 
 To get started with [Backdrop](https://backdropcms.org), clone the project repository and navigate to the project directory.
 
+=== "New projects"
+
 ```bash
-git clone https://github.com/example/example-site
-cd example-site
-ddev config
+# Create a project directory and move into it:
```

Comment:
The whole bash block needs indentation like the existing projects bash block. the missing indentation is the reason that the tabs are shown in two separate boxes.

Reply:
Good catch. Updated.